src/screens/PrivateChat/ChatList/styles.js

```diff
@@ -71,6 +71,42 @@ const styles = MediaQueryStyleSheet.create({
     flexDirection: 'row',
     justifyContent: 'flex-end',
   },
+  modalMoreContainer: {
+    flex: 1,
+    backgroundColor: 'transparent',
+    paddingLeft: 100,
+    paddingRight: 10,
+    paddingTop: Platform.OS === 'android' ? 0 : 20,
```

Comment:
It looks like a paddling for status bar, is that it? If so, you should not hardcode that, there should be a constant for it.

Reply:
fixed